docs/design/StratisSoftwareDesign.lyx

```diff
@@ -4861,18 +4967,103 @@ When encountering errors, Stratis must handle them if possible, but there
 \end_layout
 
 \begin_layout Standard
-Stratis has operation states for the pools, meta and data extend, and also
- blockdevs (listed
-\begin_inset CommandInset ref
-LatexCommand vpageref
-reference "Blockdev-states"
-plural "false"
-caps "false"
-noprefix "false"
+\begin_inset Tabular
+<lyxtabular version="3" rows="4" columns="2">
+<features tabularvalignment="middle">
+<column alignment="center" valignment="top">
+<column alignment="center" valignment="top">
+<row>
+<cell alignment="center" valignment="top" topline="true" bottomline="true" leftline="true" usebox="none">
+\begin_inset Text
+
+\begin_layout Plain Layout
+Action availability state
+\end_layout
 
 \end_inset
+</cell>
+<cell alignment="center" valignment="top" topline="true" bottomline="true" leftline="true" rightline="true" usebox="none">
+\begin_inset Text
+
+\begin_layout Plain Layout
+Description
```

Comment:
For this table, you'll have to figure out how to break lines in cells. There must be a demo in the "Naming Conventions" table.